k2/python/k2/fsa.py

```diff
@@ -18,7 +19,7 @@
 
 class Fsa(object):
 
-    def __init__(self, s: str, openfst: bool = False):
+    def __init__(self, s: str):
```

Comment:
This is a preexisting issue, but I want to remind you guys that string input won't be the normal method; more commonly I anticpate reading a tensor in from disk, e.g. N by 4 or N by 5 of int32_t.

Reply:
I was also wondering about that. What about we have
```
def from_string(...) -> 'Fsa':
```
for reading from strings, and let the default `__init__` function read from a tensor, something like
```
def __init__(self, tensor: torch.Tensor, aux_labels: Optional[torch.Tensor] = None) -> 'Fsa':
```